packages/odyssey-react-mui/src/SearchField.tsx

```diff
@@ -144,7 +143,7 @@ const SearchField = forwardRef<HTMLInputElement, SearchFieldProps>(
     });
 
     const renderFieldComponent = useCallback(
-      ({ ariaDescribedBy, id }: FieldRenderProps) => (
+      ({ ariaDescribedBy, id }) => (
```

Comment:
You'll need to type this with something like:
```
type FieldRenderProps = Partial<
  Pick<FieldComponentRenderProps, "ariaDescribedBy">
> &
  Pick<FieldComponentRenderProps, "id">;
```
Adjust if they're optional or not.